freebsd/FreeBSDProcess.c

```diff
@@ -109,6 +111,8 @@ static int FreeBSDProcess_compareByKey(const Process* v1, const Process* v2, Pro
       return SPACESHIP_NULLSTR(p1->jname, p2->jname);
    case EMULATION:
       return SPACESHIP_NULLSTR(p1->emul, p2->emul);
+   case SCHEDCLASS:
+      return SPACESHIP_NUMBER(p1->sched_class, p2->sched_class);
```

Comment:
Alphabetical sorting is probably not desireble for this SCHEDCLASS field. Better define `sched_class` as an enum and convert the field to character when displaying.

Reply:
I like this idea! :+1: